packages/roosterjs-editor-api/lib/cursor/cacheGetCursorEventData.ts

```diff
@@ -4,14 +4,22 @@ import { PluginEvent } from 'roosterjs-editor-types';
 
 const EVENTDATACACHE_CURSORDATA = 'CURSORDATA';
 
-// Read CursorData from plugin event cache. If not, create one
+/**
+ * Read CursorData from plugin event cache. If not, create one
+ * @param event The plugin event
+ * @param editor The editor instance
+ * @returns The cursor data
+ */
 export default function cacheGetCursorEventData(event: PluginEvent, editor: Editor): CursorData {
     return cacheGetEventData<CursorData>(event, EVENTDATACACHE_CURSORDATA, (): CursorData => {
         return new CursorData(editor);
     });
 }
 
-// Clear the cursor data in a plugin event
+/**
+ * Clear the cursor data in a plugin event
```

Comment:
Explain why we have this function, when to call it.

Reply:
Added